l10n_eu_product_adr_local/views/adr_goods_packaging_group_views.xml

```diff
@@ -0,0 +1,73 @@
+<odoo>
+    <record id="adr_goods_packaging_group_view_tree" model="ir.ui.view">
+        <field name="model">adr.goods.packaging.group</field>
+        <field name="arch" type="xml">
+            <tree editable="bottom">
+                <field name="name"/>
+                <field name="description"/>
+                <field name="maximum_capacity"/>
+                <field name="multiplier"/>
+            </tree>
+        </field>
+    </record>
+    <record id="adr_goods_packaging_group_view_search" model="ir.ui.view">
+        <field name="model">adr.goods.packaging.group</field>
+        <field name="arch" type="xml">
+            <search>
+                <field name="name"/>
+                <field name="description"/>
+            </search>
+        </field>
+    </record>
+    <record id="adr_goods_packaging_group_view_form" model="ir.ui.view">
+        <field name="model">adr.goods.packaging.group</field>
+        <field name="arch" type="xml">
+            <form string="Dangerous Goods Packaging Group">
+                <sheet>
+                    <group>
+                        <field name="name"/>
+                        <field name="description"/>
+                        <field name="maximum_capacity"/>
+                        <field name="multiplier"/>
+                    </group>
```

Comment:
For a new form view from scratch, a good practice is splitting it in two columns, even if model start with a few fields